pkg/cmd/completion/completion.go

```diff
@@ -1,16 +1,16 @@
-package root
+package completion
 
 import (
 	"errors"
 	"fmt"
 
 	"github.com/MakeNowJust/heredoc"
 	"github.com/cli/cli/pkg/cmdutil"
-	"github.com/cli/cli/pkg/iostreams"
 	"github.com/spf13/cobra"
 )
 
-func NewCmdCompletion(io *iostreams.IOStreams) *cobra.Command {
+func NewCmdCompletion(f *cmdutil.Factory) *cobra.Command {
```

Comment:
There is no reason why this command should receive a factory, since it's not going to do anything with that factory except access its IOStreams. So we can just provide that object directly like before.